expression/reconstructor/reconstructor.go

```diff
@@ -29,6 +31,9 @@ const (
 
 	// CodeHint hints that value should be a smart contract code, normally loaded from a file
 	CodeHint
+
+	// Bech32Hint hints that value should be a bech32 address, normally loaded from a file
+	Bech32Hint
```

Comment:
Hints represent some information about the context of a value. There is no "bech32" context, only an "address" context, there is no way of knowing just from context whether the address should be expressed as bech32 or hex.

AddressHint goes to "from" and "to" fields, for example. Bech32 is potentially in the same spots.